parity/cli/mod.rs

```diff
@@ -562,13 +565,14 @@ mod tests {
 			flag_tx_queue_gas: "auto".into(),
 			flag_tx_queue_strategy: "gas_factor".into(),
 			flag_tx_queue_ban_count: 1u16,
+			flag_tx_queue_ban_time: 180u16,
 			flag_remove_solved: false,
 			flag_notify_work: Some("http://localhost:3001".into()),
 
 			// -- Footprint Options
 			flag_tracing: "auto".into(),
 			flag_pruning: "auto".into(),
-			flag_pruning_history: 64u64,
+			flag_pruning_history: 128u64,
```

Comment:
Sorry, will move it to separate PR - I haven't changed the default, but rather fixed a bug where config value would not be used (`usage.txt` had `[default:` instead of `(default:`)
This change was only to verify that it's not working.